frontend/views/components/CreateGroupSteps/GroupRules.vue

```diff
@@ -1,95 +1,96 @@
+          :progressColor='approveColor'
+          :knobColor='approveColor'
+          circleColor='#F4F8E7'
+        )
+        i18n.subtitle(tag='p') Add Member
+
+      .c-rule
+        p.c-percent
+          span.c-pourcent-number {{ group.memberRemovalThreshold | toPercent }}
+          | %
+        circle-slider.circle-slider(
+          :value='group.memberRemovalThreshold'
+          @input='value => update("memberRemovalThreshold", value)'
+          :min='0.01'
+          :max='1'
+          :step-size='0.01'
+          :side='156'
+          :circleWidth='4'
+          :progressWidth='4'
+          :knobRadius='12'
+          :progressColor='removeColor'
+          :knobColor='removeColor'
+          circleColor='#F4F8E7'
+        )
+        i18n.subtitle(tag='p')  Remove Member
+
+    transition(name='slidedown')
+      message(
+        v-if='!superMajority'
+        severity='warning'
       )
-      i18n.subtitle(tag='p') Change Rules
-
-    .c-rule
-      p.c-percent
-        span.c-pourcent-number {{ group.memberApprovalThreshold | toPercent }}
-        | %
-      circle-slider.circle-slider(
-        :value='group.memberApprovalThreshold'
-        @input='value => update("memberApprovalThreshold", value)'
-        :min='0.01'
-        :max='1'
-        :step-size='0.01'
-        :side='175'
-        :circleWidth='4'
-        :progressWidth='4'
-        :knobRadius='12'
-        :progressColor='approveColor'
-        :knobColor='approveColor'
-        circleColor='#F4F8E7'
-      )
-      i18n.subtitle(tag='p') Add Member
-
-    .c-rule
-      p.c-percent
-        span.c-pourcent-number {{ group.memberRemovalThreshold | toPercent }}
-        | %
-      circle-slider.circle-slider(
-        :value='group.memberRemovalThreshold'
-        @input='value => update("memberRemovalThreshold", value)'
-        :min='0.01'
-        :max='1'
-        :step-size='0.01'
-        :side='175'
-        :circleWidth='4'
-        :progressWidth='4'
-        :knobRadius='12'
-        :progressColor='removeColor'
-        :knobColor='removeColor'
-        circleColor='#F4F8E7'
-      )
-      i18n.subtitle(tag='p')  Remove Member
-
-  transition(name='slidedown')
-    message(
-      v-if='!superMajority'
-      severity='warning'
-    )
-      i18n
-        | The percentage value you are choosing is most likely too low
-        | for a decision that can have a potentially significant impact
-        | on a person&rsquo;s life. Please consider using a supermajority 175.
-      | &nbsp;
-      a.link(
-        href='https://groupincome.org/2016/09/deprecating-mays-theorem/#when-majority-rule-can-harm'
-        target='_blank'
-      )
-        i18n Read more on our blog about the dangers of majority rule.
+        i18n
+          | The percentage value you are choosing is most likely too low
+          | for a decision that can have a potentially significant impact
+          | on a person&rsquo;s life. Please consider using a supermajority 175.
```

Comment:
This method of using i18n is going to make writing the parser a complete nightmare. Can you put all of this on one line please?